test/Renci.SshNet.IntegrationTests/CipherTests.cs

```diff
@@ -76,6 +76,12 @@ public void Aes256Gcm()
         {
             DoTest(Cipher.Aes256Gcm);
         }
+
+        [TestMethod]
+        public void ChaCha20Poly1305()
+        {
+            DoTest(Cipher.Chacha20Poly1305);
+        }
 #endif
```

Comment:
This doesn't need to be if-defed anymore

```suggestion
#endif
        [TestMethod]
        public void ChaCha20Poly1305()
        {
            DoTest(Cipher.Chacha20Poly1305);
        }
        
```